### ISSUE TITLE
NTR: Auscultation: single second heart sound (Cardiac & vascular)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7781]
New term request Auscultation: single second heart sound (nadea3_191206180958): S2  UMLS: 2029938

**Comment by pnrobinson** [comment]
@MickeySegal
This is not necessarily abnormal.

The second sound can remain single throughout the respiratory cycle due to either absence of one component or to synchronous occurrence of the two components. Since the pulmonary vascular impedance increases with age, many normal patients over age 50 have a single S2 or at most a narrow physiologic split on inspiration because P2 occurs early. A single second sound, however, is usually due to inability to auscultate a relatively soft pulmonic component. Such inability is rare in healthy infants, children, and young adults and is uncommon even in older persons under good auscultatory conditions using a rigid stethoscopic diaphragm.
  (https://www.ncbi.nlm.nih.gov/books/NBK341/)

Can you provide more context?

**Comment by MickeySegal** [comment]
We use this so far only for tricuspid atresia.  We have a frequency in normal of a few percent.

**Comment by pnrobinson** [comment]
@MickeySegal 
ok, adding as

Adding new term: 
- ID: HP:6000225
- label: Single second heart sound
- parents: [Abnormal second heart sound (HP:0031661)](https://hpo.jax.org/app/browse/term/HP:0031661)
- pmid: PMID:21250183
- def: The second sound can remain single throughout the respiratory cycle due to either absence of one component or to synchronous occurrence of the two components. Since the pulmonary vascular impedance increases with age, many normal patients over age 50 have a single S2 or at most a narrow physiologic split on inspiration because P2 occurs early. A single second sound, however, is usually due to inability to auscultate a relatively soft pulmonic component. Such inability is rare in healthy infants, children, and young adults and is uncommon even in older persons under good auscultatory conditions using a rigid stethoscopic diaphragm.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7781
- synonyms:
